src/mon/MgrStatMonitor.h

```diff
@@ -27,6 +28,8 @@ class MgrStatMonitor : public PaxosService {
   MgrStatMonitor(Monitor &mn, Paxos &p, const std::string& service_name);
   ~MgrStatMonitor() override;
 
+  ceph::mutex lock = ceph::make_mutex("MgrStatMonitor::lock");
```

Comment:
Is there a reason why we need to have a mutex? From what I see from the logs, it should run with 1 thread. It's not multithreaded.

Reply:
ah okay that make sense